pom.xml

```diff
@@ -2,9 +2,9 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
-  <groupId>lti</groupId>
+  <groupId>edu.umich.it.tl</groupId>
```

Comment:
With the groupid change, we need to update pom files that depend on lti-utils

Reply:
Yeah, I figured we'll have to update this version anywhere anywhere that uses it anyway. And if we have a correct groupId we can just release it to Maven Central saving some work down the road.

Reply:
The projects that are currently using this is CCM and Gdrive. I will test this fix with the CCM as it need a release soon for the infrastructure changes

Reply:
Thanks! I put this change on CCM at https://github.com/tl-its-umich-edu/canvas_course_manager/pull/249

I tested that the Dockerfile builds